README.md

```diff
@@ -24,14 +23,24 @@ The source code is located in `./src/boostvqe/.` and its composed of:
 * `plotscripts.py`: plotting functions.
 * `compiling_XXZ.py`: compilation for XXZ model.
 
-## How to run the code
+## Example
 
-For further information about the inputs:
+It follows a python snippet explaining how to run the boosting
 
-```sh
-python main.py --help
+```py
+
+from boostvqe.boost import dbqa_vqe
+from boostvqe.ansatze import hdw_efficient
+
+from qibo.models.dbi.double_bracket import DoubleBracketGeneratorType
+
+circuit = hdw_efficient(nqubits=2, nlayers=2)
+output_folder = "output"
+dbqa_vqe(circuit, output_folder, mode = DoubleBracketGeneratorType.group_commutator)
 ```
```

Comment:
here it would be nice to be able to print a report on how much was gained by running the dbqa_vqe

Reply:
I am not sure about this since this function should be called in other scripts. By the way, this PR is quite big, so this request should be handled in a separate PR.

Reply:
yep, let's move ahead towards merge without it